File: No_17.py
# 編號：CANDY-017
# 程式語言：Python
# 題目：計算數字的 2 進位裡有幾個 1
# 範例：5 -> 101 -> 2 個 1


def count_bits(num):
    return (f"{num:b}").count("1")


print(count_bits(1234))
# 5
print(count_bits(1450))
# 6
print(count_bits(9527))
# 8

File: Candy/No_17.js
// 編號：CANDY-017
// 程式語言：JavaScript
// 題目：計算數字的 2 進位裡有幾個 1
// 範例：5 -> 101 -> 2 個 1

function countBits(num) {
  return num.toString(2).replaceAll("0", "").length;
}

console.log(countBits(1234)); // 5
console.log(countBits(1450)); // 6
console.log(countBits(9527)); // 8
